airflow_clickhouse_plugin/sensors/clickhouse_sql_sensor.py

```diff
@@ -0,0 +1,23 @@
+from airflow.sensors.sql_sensor import SqlSensor
+from airflow_clickhouse_plugin.hooks import clickhouse_hook
+
+
+class ClickHouseSqlSensor(SqlSensor):
+    def __init__(
+        self,
+        clickhouse_conn_id: str = "clickhouse_default",
+        database=None,
+        *args,
+        **kwargs
+    ):
+        super(ClickHouseSqlSensor, self).__init__(conn_id=None, *args, **kwargs)
+        self._conn_id = clickhouse_conn_id
+        self._database = database
+
+    def _get_hook(self):
```

Comment:
What is this method needed for? It is not part of the `SqlSensor` interface.

Reply:
It is used in the poke method of the SqlSensor class to get a connection.

Reply:
Unfortunately it is not present in Airflow 1.10.6: https://airflow.readthedocs.io/en/1.10.6/_modules/airflow/sensors/sql_sensor.html

And we cannot use newer versions of Airflow API since [1.10.6 is the latest version supported by Google Cloud Composer](https://cloud.google.com/composer/docs/concepts/versioning/composer-versions) which is required to support for Whisk.

Please try to test your code against version 1.10.6. Thank you for your understanding.

Reply:
It seems that I need to implement poke method as it is in 1.10.6 version with some changes to get ClickHouse hook